packages/react/src/slider/root/SliderRoot.tsx

```diff
@@ -157,16 +158,14 @@ export namespace SliderRoot {
     values: ReadonlyArray<number>;
   }
 
-  export interface Props
+  export interface Props<TValue extends number[] | number = number | number[]>
```

Comment:
```suggestion
  export interface Props<Value extends readonly number[] | number = number | readonly number[]>
```
`Value` is fine ~ we don't use the `T*` convention, also the array form of value/defaultValue should be readonly arrays